data/jeos/g_cloudinit.cfg

```
@@ -63,3 +59,5 @@ runcmd:
   - ['systemctl', 'daemon-reload']
   - ['systemctl', 'enable', 'just_a_test.service']
   - ['systemctl', 'start', '--no-block', 'just_a_test.service']
+  - echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf
```

Aren't we using public ssh keys for this?

```suggestion
  - echo 'PermitRootLogin prohibit-password' > /etc/ssh/sshd_config.d/root.conf
```

---

We want to access the root with SSH, it is the same as we are doing now for sl-micro and micross.
But yes, for public cloud, we are using ssh keys for this.

---

> We want to access the root with SSH, it is the same as we are doing now for sl-micro and micross. But yes, for public cloud, we are using ssh keys for this.

Outside of the scope of this PR, but thy not also use public ssh keys here?